S3Scanner/__main__.py

```diff
@@ -137,6 +139,8 @@ def main():
     parser_group.add_argument('--buckets-file', '-f', dest='buckets_file',
                               help='Name of text file containing bucket names to check', metavar='file')
     parser_group.add_argument('--bucket', '-b', dest='bucket', help='Name of bucket to check', metavar='bucket')
+    parser_scan.add_argument('--json', '-j', dest='json_file', help='Name of file to write JSON to', nargs='?', 
+                             const='results.json', type=str, metavar='json_file')
```

Comment:
Either remove the default file name or add a note to the help string like `Defaults to results.json`